monero-outreach-docs/translations/es/merchants/merchant_faq_es.md

```diff
@@ -0,0 +1,58 @@
+# Preguntas frecuentes de los comerciantes de Monero
+
+Cuando arrancamos en Monero se nos vienen muchas preguntas a la cabeza. Hemos contestado algunas de las más comunes y nos gustaría saber que otras podrían haber y responderlas para que otros se beneficien.
+
+### _Si acepto Monero, ¿se incrementarán mis ventas?_
+
+De pronto, lo más probable es que sí, pero no es algo seguro. Hay una gran parte de la comunidad de Monero que cree firmemente en el respaldo a los comerciantes que aceptan Monero. Los comerciantes tienen la manera de hablar con los clientes de cómo se protege su privacidad al hablar de Monero con ellos. También depende de que tan bueno se sea en informar al cliente de lo que se está haciendo. Historicamente aceptar criptomonedas tiende a pagar bien, Coindesk hizo una encuesta en el 2014, [coindesk.com/survey-results-one-fourth-of-bitcoin-merchants-enjoy-10-sales-boost](https://www.coindesk.com/survey-results-one-fourth-of-bitcoin-merchants-enjoy-10-sales-boost), que encontró que el 25% de los nuevos comerciantes de Bitcoin vieron un incremento del 10% en sus ventas. Una encuesta un poco más reciente realizada en el 2018 por CreditCoin, [prnewswire.com/news...study-shows-complicated-purchasing-process-is-biggest-hurdle-for-cryptocurrencies...html](https://www.prnewswire.com/news-releases/new-study-shows-complicated-purchasing-process-is-biggest-hurdle-for-cryptocurrencies-300674527.html), encontró que el 75% de los que tienen criptomonedas les gustaría gastar sus monedas en tiendas físicas. Además, si tienes una plataforma de ventas internacional fácil de ejecutar y con comisiones bajas, puedes esperar un mayor número de clientes si tu negocio es digital o de venta por correo.
+
+### _¿Qué diferencia hay entre aceptar Monero a aceptar Bitcoin?_
+
+La configuración tanto de Bitcoin como de Monero son similares, pero las cadenas de bloques son diferentes. Monero tiene algunas características únicas que le podrían interesar más a los comerciantes que son:
+
+- **Pagos más rápidos** - Monero tiene un tiempo de verificación de 2 minutos por bloque a diferencia de los 10 minutos por bloque en Bitcoin. En promedio, tu transacción llegaría 5 veces más rápido a la cadena de bloques de Monero que a la de Bitcoin.
+- **Menos costoso** - Monero tiene comisiones mucho más bajas que Bitcoin y además, Monero continúa siendo cada vez más eficiente, la tendencia en términos de comisiones es tener comisiones más bajas que las de Bitcoin.
+- **No hay retrasos en las transacciones** - Los bloques de Monero son dinámicos mientras que Bitcoin tiene bloques fijos. Cuando la red de Bitcoin está ocupada y los bloques están llenos, las transacciones deben, o pagar comisiones altas, o retrasarse.
```

Comment:
Instead of "Monero continúa siendo cada vez más eficiente, la tendencia en términos de comisiones es tener comisiones más bajas que las de Bitcoin.", I'd change to:  "Monero continúa siendo cada vez más eficiente y  la tendencia es tener comisiones cada vez mas bajas. "

Reply:
Changed :).